lib/generators/meetup_client/install_generator.rb

```diff
@@ -0,0 +1,11 @@
+module MeetupClient
+  module Generators
+    class InstallGenerator < Rails::Generators::Base
+      source_root File.expand_path('../templates', __FILE__)
+      desc 'This generator creates an initializer file at config/initializers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.